README.md

```diff
@@ -1,2 +1,74 @@
 # Ruby-Tic-Tac-Toe
-Tic-Tac-Toe Game 
\ No newline at end of file
+ Tic-Tac-Toe is a game for two players, X and O, who take turns marking the spaces in a 3×3 grid. The player who succeeds in placing three of their marks in a horizontal, vertical, or diagonal row is the winner.
+## Built With
+
+- Ruby
+
+## Getting Started
+
+## Instructions to play the Game
+
+- Two players are required to play the game.
+- The first player is X and the second player is O.
+- The players are required to input numbers between 0 - 8, to pick a position in the board.
+- If the player pick a position that is occupied, the player will be required to pick another position.
+- The first player to get identical letter in the row or column or left and right diagonal wins the game.
+- If no identical letter occurs and the board is full, the game will end in a draw
+
+## How to run the game in your terminal
+
+- Run this command line: bin/main.rb
+
+**This is an example of how you may give instructions on setting up your project locally.**
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- Install ruby on your local machine
+
+### Setup
+
+- Clone the repository on your local machine
+- cd into the folder, find the bubble_sort file and open it
+
+### Install
+
+- Install Ruby (most recents versions)
+- Install VSCode or any code editor you like
+- Add gem 'rubocop' to Gemfile (not sure how to use Gemfile? Read <a href="https://bundler.io/v1.15/guides/bundler_setup.html">this</a>).
+- Run bundle install.
+
+### Run tests
+
+- Add a copy of <a href="https://github.com/microverseinc/linters-config/blob/master/ruby/.stickler.yml">.stickler.yml</a> and <a href="https://github.com/microverseinc/linters-config/blob/master/ruby/.rubocop.yml">.rubocop.yml</a> to root directory.
+- Run rubocop to fix linters errors
+
+
+## Authors
+
+👤 Adewale Adeshola
+
+- Github: [@githubhandle](https://github.com/Eshy10)
+- Twitter: [@twitterhandle](https://twitter.com/AdesholaAdewal6?s=09)
+- Linkedin: [linkedin](https://www.linkedin.com/in/adewale-adeshola-b0b581139/)
+
+👤 Roman Nikolaev 
+
+- Github: [@githubhandle](https://github.com/vzdrizhni)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- Linkedin: [linkedin](https://twitter.com/metabruta)
+
+## 🤝 Contributing
+
+Contributions, issues and feature requests are welcome!
+
+Feel free to check the [issues page]().
```

Comment:
- [ ] right now the issues page link leads to an empty page, add the URL within the parenthesis to the issues page link or remove it from the Readme file